diff --git a/node_modules/@expo/metro-config/build/transform-worker/getAssets.js b/node_modules/@expo/metro-config/build/transform-worker/getAssets.js
index 70db7bd..d76647f 100644
--- a/node_modules/@expo/metro-config/build/transform-worker/getAssets.js
+++ b/node_modules/@expo/metro-config/build/transform-worker/getAssets.js
@@ -120,7 +120,7 @@ async function ensureOtaAssetHashesAsync(asset) {
     if (assetPathQueryParameter && assetPathQueryParameter[2]) {
         const assetPath = assetPathQueryParameter[2];
         // @ts-expect-error: marked as read-only
-        asset.httpServerLocation = asset.httpServerLocation.replace(assetPath, encodeURIComponent(assetPath));
+        asset.httpServerLocation = asset.httpServerLocation.replace(assetPath, encodeURI(assetPath));
     }
     return asset;
 }
